md/BonitaBPM_platform_setup.md

```diff
@@ -50,14 +57,14 @@ It pushes the configuration from the `platform_conf/current` folder into the dat
 
 Before running it, the tool must be configured to point to the database of the Bonita BPM Platform.
 
-To do so, after you have created the database, modify the tool `database.properties` file:  
+To do so, after you have created the database, modify the tool `database.properties` file:
 Set the right db vendor by commenting and uncommenting properties inside this file and change connection url, user credentials, database name and so on.
 
-If you are using an Oracle or Microsoft SQL Server database, you will need to add the related JDBC driver in the `lib` folder.
+If you are using an Oracle or Microsoft SQL Server database, you will need to add the related JDBC driver in the `lib` folder. (H2, MySQL and PostgreSQL drivers are already shipped in the tool).
```

Comment:
use imperative (I know you just changed the end of the sentence but while we are here :-) )